installation/cmd/run.sh

```diff
@@ -188,31 +161,6 @@ if [[ -n ${API_METADATA_VALIDATOR_IMAGE} ]]; then
   set_api_metadata_validator_image true $API_METADATA_VALIDATOR_IMAGE "http://localhost"
 fi
 
-if [[ -z ${OIDC_HOST} || -z ${OIDC_CLIENT_ID} ]]; then
```

Comment:
This whole block should not be removed as the JWT flows will never work locally. The overrides found in `$PATH_TO_COMPASS_OIDC_CONFIG_FILE` could be scoped down only to `idpHost` and they should be overwritten to the new location `.global.oathkeeper...`